handlers/istio_config.go

```diff
@@ -304,8 +303,24 @@ func IstioConfigCreate(w http.ResponseWriter, r *http.Request) {
 		RespondWithError(w, http.StatusBadRequest, "Create request could not be read: "+err.Error())
 	}
 
-	business.IstioConfig.CreateIstioConfigDetail(api, namespace, objectType, objectSubtype, object, string(body))
+	json, err := business.IstioConfig.ParseJsonForCreate(objectType, objectSubtype, body)
```

Comment:
Doesn't this kinda ruin the separation? Shouldn't it be a detail of Create?

Reply:
I thought that handler layer should process the input parameters and validate those are correct.
Return a BadRequest / 400 error, like a malformed or bad body.
Yeah, this is something in the middle of handlers / business layer.
Return a 404 from the business layer ? and add a IsBadRequest() after the CreateIstioConfigDetail() ?

I can do that if it's more clear.

Reply:
I think it would make more sense, given that we return 404 from business layer also later. Thus, the handler should probably only verify the JSON can be extracted and not care about the actual data.

Reply:
Ok, will move it inside the business and also check the 400 in the returning error, that will place same logic together.

Reply:
@burmanm I've sent a commit to move this logic, let me know if the PR works for you.